build.sbt

```diff
@@ -77,7 +77,8 @@ lazy val root = project
     clientJS,
     tools,
     codegenSbt,
-    federation
+    federation,
+    zioHttp
```

Comment:
Super nitpick but do you mind putting it after `play`, just to keep things in order 😄 
One more thing: you need to add it in .circleci/config.yml where the others are.